src/components/TimeoutModal/TimeoutModal.js

```diff
@@ -10,14 +10,14 @@ const TimeoutModal = () => {
 	const [isModalOpen, setIsModalOpen] = useState(false);
 	const { auth, setAuth } = useAuth();
 	const { setModalTimeout } = useTimeout();
-	let fullTimeoutDuration = auth.sessionTimeout;
+	// let fullTimeoutDuration = auth.sessionTimeout;
```

Comment:
If we dont need this variable, we might want to remove them from the file and keep the code clean.

Reply:
Agreed, I can do that real quick